usmqe_tests/conftest.py

```diff
@@ -499,3 +499,20 @@ def fill_memory():
         SSH[host].run(teardown_cmd)
         return request.param
     return measure_operation(fill_memory)
+
+
+@pytest.fixture()
+def gluster_volume(request):
+    """
```

Comment:
I would start with a note: "Use this fixture when a test case needs at least one gluster volume".